utils/cursor_md_exporter.py

```
@@ -0,0 +1,146 @@
+            session_meta = {}
+
+    session_id: str = session_meta.get("agentId", db_path.parent.name)
+    created_ms: int = session_meta.get("createdAt") or int(datetime.now().timestamp() * 1000)
+    session_name: str = session_meta.get("name") or f"Session {session_id[:8]}"
+    mode: str = session_meta.get("mode", "")
+
+    # Reconstruct conversation.
+    try:
+        messages = traverse_blobs(str(db_path))
+        bubbles = messages_to_bubbles(messages, created_ms)
+    except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)